### PR TITLE
Add SwiftLint and update source code to match rules

### PR DESCRIPTION
**Pull request by pichfl**
Right now this fails the build with 15 remaining issues.

- [x] `force_cast` in DataManager L63
- [x] `force_cast` in PlayerViewController L202
- [x] `force_cast` in PlayerViewController L209
- [x] `force_cast` in PlayerViewController L216
- [x] `force_cast` in PlayerViewController L226
- [x] `force_cast` in PlayerViewController L237
- [x] `force_cast` in PlayerViewController L252
- [x] `force_cast` in ChaptersViewController L202
- [x] `force_try` in AppDelegate L33
- [x] `force_try` in AppDelegate L59
- [x] `force_cast` in ListBooksViewController L245
- [x] `force_try ` in ListBooksViewController L291
- [x] `force_cast` in ListBooksViewController L337
- [x] `force_try ` in PlayerManager L288
- [x] `force_try ` in PlayerManager L297

Warnings from rules `function_body_length` and `file_length` exist for these files:

- ListBooksViewController
- PlayerManager
- PlayerViewController

I'm still rather new to Swift, so it would be nice to have some input regarding the remaining issues. I'll try to tackle the disabled rules by extracting functionality from the listed classes into separate functions/files where appropriate. I see a lot of potential for this especially when planning for upcoming changes.

### CODE REVIEW

File: Audiobook Player/PlayerManager.swift [hunk 1]
```diff
@@ -10,183 +10,180 @@ import Foundation
...
+
     //timer to update labels about time
-    var timer:Timer!
-    
+    var timer: Timer!
+
     func isLoaded() -> Bool {
-        guard let _ = self.audioPlayer else {
-            return false
+        if self.audioPlayer != nil {
```
Comment (GianniCarlo) [inline Audiobook Player/PlayerManager.swift]: let's do `return self.audioPlayer != nil`

**Comment by GianniCarlo** [comment]
nice! thanks for adding a linter to the project 👌 I'll continue checking it later and throughout this week

File: Audiobook Player/PlayerManager.swift (hunk 1 above)
Comment (pichfl) [reply Audiobook Player/PlayerManager.swift]: Oh I wasn't sure that was an option and I didn't try 😄 will update.

**Comment by pichfl** [comment]
😱 I just managed to complete all the compile errors that keep the project from building. I left the warnings enabled as they will most likely go away when we work on the upcoming tasks. 

I also made a small bugfix which fixes a simple double invocation that did no harm but caused a warning.

**Comment by GianniCarlo** [comment]
nice! I'll test it out later today 💪

File: Audiobook Player/PlayerViewController.swift [hunk 2]
```diff
@@ -171,132 +171,136 @@ class PlayerViewController: UIViewController {
...
+            PlayerManager.sharedInstance.setChapter(chapter)
+        }
     }
-    
-    @IBAction func didSelectSpeed(_ segue:UIStoryboardSegue){
-        
+
+    @IBAction func didSelectSpeed(_ segue: UIStoryboardSegue) {
+
         guard PlayerManager.sharedInstance.isLoaded() else {
```
Comment (GianniCarlo) [inline Audiobook Player/PlayerViewController.swift]: in this guard, let's add the check of `let viewController = segue.source as? SpeedViewController`

File: Audiobook Player/PlayerViewController.swift [hunk 3]
```diff
@@ -171,132 +171,136 @@ class PlayerViewController: UIViewController {
...
+            self.speedButton.setTitle("Speed \(String(PlayerManager.sharedInstance.currentSpeed))x", for: UIControlState())
+        }
     }
-    
-    @IBAction func didSelectAction(_ segue:UIStoryboardSegue){
-        
+
+    @IBAction func didSelectAction(_ segue: UIStoryboardSegue) {
+
         guard PlayerManager.sharedInstance.isLoaded() else {
```
Comment (GianniCarlo) [inline Audiobook Player/PlayerViewController.swift]: same thing here

**Comment by pichfl** [comment]
@GianniCarlo Extracted your comments into #49

**Pull request merged**